roles/openshift_storage_glusterfs/tasks/glusterfs_config.yml

```diff
@@ -9,6 +9,7 @@
     glusterfs_use_default_selector: "{{ openshift_storage_glusterfs_use_default_selector }}"
     glusterfs_storageclass: "{{ openshift_storage_glusterfs_storageclass }}"
     glusterfs_storageclass_default: "{{ openshift_storage_glusterfs_storageclass_default | bool }}"
+    glusterfs_storageclassblock_default: "{{ openshift_storage_glusterfs_storageclassblock_default | bool }}"
```

Comment:
Rename to `glusterfs_block_storageclass_default` and move it down with the other glusterblock variables.